components/Optional/index.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import { useBlockEditContext } from '@wordpress/block-editor';
+
+export const Optional = ({ value, children }) => {
+	const { isSelected } = useBlockEditContext();
+	return !!value && isSelected && <>{children}</>;
```

Comment:
```suggestion
	return !!value && isSelected && children;
```

I think we can omit the fragment here :)